src/TestFramework/AbstractTestFrameworkAdapter.php

```diff
@@ -229,20 +217,21 @@ public function getInitialTestsFailRecommendations(string $commandLine): string
      *
      * @return string[]
      */
-    private function findPhp(bool $includeArgs = true): array
+    private function findPhp(): array
     {
-        if ($this->cachedPhpPath === null || $this->cachedIncludedArgs !== $includeArgs) {
-            $this->cachedIncludedArgs = $includeArgs;
-            $phpPath = (new PhpExecutableFinder())->find($includeArgs);
+        $phpExec = (new PhpExecutableFinder())->find(false);
```

Comment:
`$this->cachedPhpPath` property was required. I've just thought about renaming.

We don't want to execute `find` more then once. Fine with me if it'll just check on non empty.